addons/website/views/snippets/s_social_media.xml

```diff
@@ -32,7 +32,8 @@
             <we-list string="Social Networks" t-att-data-add-item-title="add_item_title"
                 data-render-list-items="" data-has-default="multiple" data-name="social_media_list"
                 data-default-value="https://www.example.com" data-id-mode="name"
-                data-new-elements-not-toggleable="true" data-no-preview="true"/>
+                data-new-elements-not-toggleable="true" data-no-preview="true"
+                data-render-on-input-blur="true"/>
```

Comment:
I tested the "Social Media" and it seems that changing the values in the inputs has no effect?
(Except for custom links). Also, when moving an item with the "o_we_drag_handle" icon, there is an issue with "input" widths. I know all that is not related to your change, but it's worth fixing it in the same task (check version), since it is "we-list" bugs.
Screenshot =>
![Capture d’écran de 2023-01-04 11-29-59](https://user-images.githubusercontent.com/52911687/210536293-df17732a-d140-48fd-ae7d-ece5c33a15f0.png)

Reply:
> Are you saying that because you don't see any change in the DOM? The url of social media (not custom) are stored in DB, that's the reason.

I tested the link directly after modifying it, but you have to save the page first. So no worries..

Reply:
> Indeed that is an issue. What do you think, should we fix it in stable or only on master (as it does not break any flow) ?

IMO, I would fix in stable because it's a bit ugly, if possible it's always better.

Reply:
OK, I will create a task for that :+1: